cla_public/config/common.py

```diff
@@ -11,6 +11,8 @@
 
 CLEAR_SESSION = True
 
+GOV_UK_START_PAGE = 'https://www.gov.uk/check-legal-aid'
```

Comment:
This will sound extremely :neckbeard:y and we don't need to solve this now.

Do we need to redirect to https://www.gov.uk/gwirio-os-ydych-yn-gymwys-i-gael-cymorth-cyfreithiol (Welsh) if our locale is set to Welsh?

Reply:
I thought about that but then thought it might be more trouble than it's worth right now. It introduces another coupling between the two systems.

If we wanted to do this well, we'd want to be alerted if either of these link changes. Or check if there's an API where we can get a list of the start pages for an application.